Info-Orbs/src/main.cpp

```diff
@@ -40,7 +40,7 @@ bool tft_output(int16_t x, int16_t y, uint16_t w, uint16_t h, uint16_t *bitmap)
 
 ScreenManager* sm;
 WidgetSet* widgetSet;
-StockScreenWidget *stockScreenWidget;
+//StockScreenWidget* stockScreenWidget;
```

Comment:
I think this can be fully removed instead of just commented out

Reply:
I agree